test/suite/test_verify_disagg.py

```diff
@@ -76,7 +76,7 @@ def verify(self, sessions, expected_error = None):
                 self.assertRaisesException(wiredtiger.WiredTigerError, \
                     lambda: session.verify(self.uri), os.strerror(expected_error))
             else:
-                session.verify(self.uri)
+                self.verifyUntilSuccess()
```

Comment:
Should you give the `session` here as we are not using `self.session` originally?
```suggestion
                self.verifyUntilSuccess(session)
```

Reply:
Good catch!